### PR TITLE
removed javascript from title #477

### LINKED ISSUE #477: Use of "JavaScript" and articles in titles
I'm new here, but when I look at the list of JavaScript articles, several of their titles lead with "JavaScript...". This creates a confusing list as it seems to be out of alphabetical order, with entries starting with "j" sprinkled throughout. I would suggest not putting "JavaScript" in the title as it is redundant because we are in the JavaScript folder.

This may have also lead to some accidental duplication, for example there is an entry for "JavaScript for Loop" and "For Loops". Also there is "For in" and "JavaScript Forin Loop".

Additionally, I would suggest avoiding starting titles with articles (a, an, the), as in "The JavaScript Continue Statement" and "The Random Method". I would think that the key word of the article (that being defined) should be the first word in the title to create a more visually comprehensible list.

This is the list of titles as it stands now on the live site:

```
Advantages and Disadvantages of JavaScript
Angularjs Interview Questions
JavaScript Arguments
Arithmetic Operation JavaScript
JavaScript Assignment Operators
JavaScript Boolean
JavaScript Break Statement
JavaScript Callback Functions
Classes in JavaScript
JavaScript Closures
JavaScript Code Block
Code Linting in JavaScript
Comparison and Logical Operators
JavaScript Comparison Operators
JavaScript Loops
Conditional Ternary Operators
The JavaScript Continue Statement
Converting Strings to Numbers
JavaScript Dowhile Loop
Error Handling and Try Catch Throw
JavaScript Falsy Values
For in
JavaScript for Loop
For Loops
JavaScript for of Loop
JavaScript Forin Loop
Form Validation
Function Closures
Function Invocation
JavaScript Functions List
JavaScript Get Timestamp
JavaScript Global Variables
Higher Order Functions
HTML DOM getElementById Method
HTML Dom Innerhtml Property
If Else Statements
The JavaScript Ifelse Statement
JavaScript Immutable Types
JavaScript Isloggedinasync
Location Object
Location Reload Method
Loops and Iteration
Manipulate JavaScript Objects
Manipulating Cookies
Multiply Two Numbers with JavaScript
Naming Convention for JavaScript
Numbers
Onclick Event
Onload Event
Operators
Output
Popup Boxes
Promises
Prototypes
Regular Expressions Reference
The JavaScript Return Statement
Scopes in JavaScript
Semicolons in JavaScript
JavaScript Strict Mode
Switch Statements
JavaScript Ternary Operator
The JavaScript Global Object
The HTML Dom
JavaScript Truth Table
The Random Method
this reference in JavaScript
Timing Events
JavaScript Truthy Value
Typeof
The JavaScript Undefined Primitive Value
Where to
JavaScript While Loop
Window Confirm Method
Window Location
Window Open Method
Window Setinterval Method
Window Settimeout Method
With
```

### PR DESCRIPTION
**Pull request by dhanushuUzumaki**
Closes #477

### CODE REVIEW

File: src/pages/javascript/tutorials/add-new-properties-to-a-object/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Add New Properties to a JavaScript Object
+title: Add New Properties to a Object
```
Comment (systimotic) [inline src/pages/javascript/tutorials/add-new-properties-to-a-object/index.md]: an Object

File: src/pages/javascript/tutorials/the-equivalent-of-jquerygetjson/index.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: JavaScript Version of Jquerygetjson
+title:  Equivalent of Jquerygetjson
```
Comment (systimotic) [inline src/pages/javascript/tutorials/the-equivalent-of-jquerygetjson/index.md]: There's an extra space here.

File: src/pages/javascript/tutorials/what-does-void-0-mean/index.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 ---
-title: What Does JavaScript Void 0 Mean
+title: What Does Void 0 Mean
```
Comment (systimotic) [inline src/pages/javascript/tutorials/what-does-void-0-mean/index.md]: I think this title should remain the same. This is talking about links with `href="javascript:void(0)"`. Maybe the title should be changed to "What javascript:void(0) means"

File: src/pages/visual-design/visual-hierarchy/index.md [hunk 4]
```diff
@@ -11,4 +11,5 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 
 #### More Information:
 <!-- Please add any articles you think might be helpful to read before writing the article -->
-- <a href='https://www.interaction-design.org/literature/article/visual-hierarchy-organizing-content-to-follow-natural-eye-movement-patterns' target='_blank' rel='nofollow'>www.interaction-design.org: Visual Hierarchy: Organizing content to follow natural eye movement patterns</a>
+
```
Comment (systimotic) [inline src/pages/visual-design/visual-hierarchy/index.md]: Can you add the link in this file back in?

File: src/pages/computer-science/what-is-an-api/index.md [hunk 5]
```diff
@@ -1,15 +1,15 @@
 ---
-title: What Is an API
+title: What Is an Api
```
Comment (systimotic) [inline src/pages/computer-science/what-is-an-api/index.md]: Can you undo the changes in this file?

Comment (dhanushuUzumaki) [reply src/pages/computer-science/what-is-an-api/index.md]: This is not a change I made. I found the reason to be the pre-commit normalization. I changed it to API. After committing, it is back to API and the file is unstaged. Should this be opened as an issue?

Comment (dhanushuUzumaki) [reply src/pages/computer-science/what-is-an-api/index.md]: @systimotic I found the reason to be the titleify function. Since 'API' is not in preformatted words list, it is making it 'Api'.

File: src/pages/javascript/tutorials/using-anonymous-functions-for-private-namespacing-in-your-apps/index.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Using Anonymous Functions for Private Namespacing in Your JavaScript Apps
+title: Using Anonymous Functions for Private Namespacing in Your Apps
```
Comment (systimotic) [inline src/pages/javascript/tutorials/using-anonymous-functions-for-private-namespacing-in-your-apps/index.md]: I'm sorry for requesting a change to something that was already like this the last time I reviewed. Can you change this to "Using Anonymous Functions for Private Namespacing"?

File: src/pages/javascript/tutorials/the-equivalent-of-jquerygetjson/index.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ---
-title: JavaScript Version of Jquerygetjson
+title: Equivalent of Jquerygetjson
```
Comment (systimotic) [inline src/pages/javascript/tutorials/the-equivalent-of-jquerygetjson/index.md]: Can you remove "the" from the filename?

File: src/pages/javascript/tutorials/how-to-create-a-dropdown-menu/index.md [hunk 8]
```diff
@@ -1,9 +1,9 @@
 ---
-title: How to Create a Dropdown Menu with CSS and JavaScript
+title: How to Create a Dropdown Menu
```
Comment (systimotic) [inline src/pages/javascript/tutorials/how-to-create-a-dropdown-menu/index.md]: I think you can remove this article instead of renaming it, it's not relevant to this section.

File: src/pages/javascript/tutorials/delete-properties-from-a-object/index.md [hunk 9]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Delete Properties from a JavaScript Object
+title: Delete Properties from a Object
```
Comment (systimotic) [inline src/pages/javascript/tutorials/delete-properties-from-a-object/index.md]: "an Object" too. Sorry, missed this last time.

File: src/pages/computer-science/what-is-an-api/index.md (hunk 5 above)
Comment (systimotic) [reply src/pages/computer-science/what-is-an-api/index.md]: @dhanushuUzumaki Yep, that's correct, good found! Can you add it to the `preformatted` array?

File: src/pages/javascript/tutorials/delete-properties-from-a-object/index.md (hunk 9 above)
Comment (dhanushuUzumaki) [reply src/pages/javascript/tutorials/delete-properties-from-a-object/index.md]: No need for a sorry. Thanks for reviewing. 😄

File: src/pages/javascript/tutorials/how-to-create-a-dropdown-menu/index.md (hunk 8 above)
Comment (dhanushuUzumaki) [reply src/pages/javascript/tutorials/how-to-create-a-dropdown-menu/index.md]: There are a bunch of how-to articles like this. Should I remove them all?

Comment (systimotic) [reply src/pages/javascript/tutorials/how-to-create-a-dropdown-menu/index.md]: @dhanushuUzumaki Maybe, can you list some of the ones that don't belong? Maybe we should look at moving them to the correct place instead. That's out of the scope of this PR though, we can do that in a different one.

Comment (dhanushuUzumaki) [reply src/pages/javascript/tutorials/how-to-create-a-dropdown-menu/index.md]: Sure. I will just address these comments and push to this one and create a new issue to move the how-to articles.

**Comment by erictleung** [comment]
@systimotic @dhanushuUzumaki it seems that this PR has diverged too much and I can't resolve these merge conflicts from GitHub. I would hate to see so much work go to waste. This is a large change. Let us know if you're willing to take a stab at fixing these commits or just making a new PR. Thanks!

**Comment by dhanushuUzumaki** [comment]
@erictleung Sorry. Got tied up at work and wasn't able to look into it. I will try to look into it this weekend. If I can solve it, I will update the PR else let me create a new one.

**Comment by johnkennedy9147** [comment]
@dhanushuUzumaki closing PR as no activity since your comment 16 days ago. You put a lot of work into this so feel free to reopen this, or open a fresh PR if you resolve the issues. thanks
